src/getScreenshots.test.js

```diff
@@ -66,4 +70,59 @@ describe('gets Screenshots', () => {
       ]);
     });
   });
+
+  it('Runs the onBeforeSuiteScript once before all scenarios', () => {
+    class MockSnapshotter {
+      takeSnap() {
+        return jest.fn().mockImplementation(() => Promise.resolve());
+      }
+    }
+    const executeScriptMock = jest
+      .fn()
+      .mockImplementation(() => Promise.resolve());
+    executeScript.mockImplementation(executeScriptMock);
```

Comment:
I'm not sure this is testing quite what you want.
could we maybe test something like that execute script has been called with `'./src/__mocks__/onBeforeSuiteMock.js` or something similar